trunion/__init__.py

```diff
@@ -33,9 +41,26 @@ def includeme(config):
         raise Exception("No issuers provided in the config file!")
     config.registry.settings['trunion.permitted_issuers'] = iss
 
+# Work around for WEIRD behaviour seen with an attempt to upgrade to mozsvc 0.8
```

Comment:
Yes.  The workarounds are for the now-missing features in the newer versions of mozsvc.